ArcGISRuntimeSDKQt_QMLSamples/Scenes/AddIntegratedMeshLayer/AddIntegratedMeshLayer.qml

```diff
@@ -16,13 +16,16 @@
 
 import QtQuick 2.6
 import Esri.ArcGISRuntime 100.12
+import QtQuick.Dialogs 1.2
```

Comment:
same comment here - I suggest using the QtQuickControls 2 dialog instead and removing this dependency